lib/transformer.dart

```diff
@@ -383,7 +384,7 @@ $initializersBuffer
   /// [lib]. This includes exported methods from other libraries too.
   List<FunctionElement> _topLevelMethodsOfLibrary(
       LibraryElement library, Set<LibraryElement> seen) {
-    var methods = [];
+    List<FunctionElement> methods = [];
```

Comment:
You should be able to just do `var methods = <FunctionElement>[]`, which is actually more correct in strong mode than this (which has an implicit cast from List<dynamic> to List<FunctionElement> afaik).